http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -885,6 +885,45 @@ paths:
               schema:
                 $ref: "#/components/schemas/ErrorReport"
 
+  "/api/v2/runtimes/{workspaceId}/deleteAll":
+    post:
+      summary: Deletes ALL runtimes associated with the given workspace Id
+      description: deletes all runtimes in a workspace
+      operationId: deleteAllRuntimesV2
+      tags:
+        - runtimes
+      parameters:
+        - in: path
+          name: workspaceId
+          description: workspaceId
+          required: true
+          schema:
+            type: string
+        - in: query
+          name: deleteDisk
+          description: Whether or not the disk associated with the apps should be deleted. Default to false if not specified.
+          required: false
+          schema:
+            type: boolean
+            default: false
+      responses:
+        "202":
+          description: Runtime(s) deletion request accepted
```

Comment:
So this call is asynchronous, correct? That means that we will not have a way of knowing when/if the deletion ultimately succeeds. What is the suggestion for us to determine that? Should we poll WSM to see if resources still exist, or poll a getRuntime method? It would be great if there was a way we could poll the deletion itself (like an job control ID being returned) so that if it fails, we can present the user with the cause of failure.

Reply:
You should be able to call the `ListRuntimes` method for that workspace to see the statuses @cahrens

Reply:
OK, we also have a few other questions, so I scheduled a meeting for Workspaces and IA to touch base.